public/swagger.v1.json

```diff
@@ -969,15 +969,12 @@
           "x-go-name": "OpenIssues"
         },
         "owner": {
-          "x-go-name": "Owner",
           "$ref": "#/definitions/User"
```

Comment:
@richmahn Why did you change this file? It's my understanding that file is automatically generated